lib/elixir/lib/kernel.ex

```diff
@@ -1669,14 +1672,14 @@ defmodule Kernel do
 
   """
   @spec inspect(Inspect.t, Keyword.t) :: String.t
-  def inspect(arg, opts \\ []) when is_list(opts) do
-    opts  = struct(Inspect.Opts, opts)
-    limit = case opts.pretty do
-      true  -> opts.width
+  def inspect(arg, options \\ []) when is_list(options) do
+    options = struct(Inspect.Opts, options)
+    limit = case options.pretty do
+      true  -> options.width
```

Comment:
I would love to push for that yeah. But places like this are "more important" as users get to see them in the documentation IMO.